gnome-initial-setup/gis-driver.h

```diff
@@ -71,8 +71,7 @@ void gis_driver_add_page (GisDriver *driver,
 
 void gis_driver_save_data (GisDriver *driver);
 
-GisDriver *gis_driver_new (GisDriverMode mode,
-                           gboolean      inhibit_idle);
+GisDriver *gis_driver_new (GisDriverMode mode);
```

Comment:
Do any corresponding polkit rules files need to be removed here as well? (I only ask because I had to override some polkit rules in order to make inhibit idle work on a different project).

Reply:
No, this inhibits idle by talking to gnome-session. System services talk directly to logind to inhibit idle, and I see there is a polkit action for that.

Reply:
Makes sense